src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -1550,7 +1562,9 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
           {maybeAlert}
           {maybeSymAfterRuneAlert}
 
-          <Button
+          <ButtonWalletPredicate
+            isValidWallet={!isDemoWallet}
```

Comment:
Same q: does it mean that any wallet that's not native demo will be considered a "valid wallet", even if it doesn't support the chain?

Reply:
https://github.com/shapeshift/web/pull/7270/files#r1661979337